### ISSUE TITLE
"Incorrect Password" error message potentially identifies a valid `Passphrase` has been entered

### ISSUE THREAD

**Issue by S-Coyle** [maidsafe/safe-mobile-browser#235]

**Comment by ravinderjangra** [comment]
@S-Coyle should we just show invalid credentials dialog here? or do you have something else in mind?

**Comment by S-Coyle** [comment]
@ravinderjangra Yes, I was thinking a message along the lines of “Login details not found, please try again”

**Comment by ravinderjangra** [comment]
Showing something related to the "correct network" may confuse some users so I think we can keep this simple. I can raise a PR if that's fine.

**Comment by ravinderjangra** [comment]
@S-Coyle I'm moving this issue to the authenticator repo since It's related to the auth app.
